command/pr.go

```diff
@@ -538,11 +542,9 @@ func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
 			} else if reviews.Approved {
 				fmt.Fprint(w, utils.Green("✓ Approved"))
 			}
-		} else {
-			fmt.Fprintf(w, " - %s", prStateTitleWithColor(pr))
```

Comment:
This was moved so the non-open status can be shown on the title line, before the URL.